bottomless/src/replicator.rs

```diff
@@ -935,6 +935,12 @@ impl Replicator {
         })?;
         tracing::trace!("Local change counter: {change_counter}");
 
+        // TODO: we shouldn't leak the connection here but for some reason when this connection get
+        // dropped it seems to checkpoint the database
+        if std::env::var("LIBSQL_DISABLE_INIT_CHECKPOINTING").is_ok() {
+            std::mem::forget(conn);
```

Comment:
Is this safe? @penberg @sivukhin May I borrow your brains here? Isn't leaking sqlite connection leading to some disaster over time?